index.html

```diff
@@ -1,19 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Karma</title>
+  <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+  <header>
+    <img class="img" src="./img/karma-logo.svg" alt="Logo"></img>
+    <nav class="logo1">
+      <ul class="list">
+        <li><a class="bar" href="index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a href="store.html">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+    </nav>
+  </header>
+<main>
+  <div class="karma">
+    <h1> Introducing karma <h1>
+    <p> Bring WiFi with you, everywhere you go.<p>
+    <button><a href="#">Learn More</a></button>  
+  </div>  
+</main>
+<p>Everyone needs a little Karma</p>
+<article>
+  <div>
+    <img src="./img/icon-devices.svg" alt="devices">
+    <h5>Internet for all devices</h5>
+  </div>   
+  <div>
+    <img src="./img/icon-coffee.svg" alt="coffee">
+    <h5>Boots your productivity</h5>
+  </div>
+  <div>
+    <img src="./img/icon-refill.svg" alt="refill">
+    <h5>Pas as You Go</h5>
```

Comment:
Excellent job Pamela, I am impressed with your code, and I would like to suggest that in lines 37, 41, and 45, try to use a '<p>' or span instead of 'h5'. about the rest, it's all clear. Well done!